stereo_image_proc/cfg/Disparity.cfg

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env python
```

Comment:
Do we still need dynamic reconfigure in ros2 ?
The new parameter api support runtime changes to parameter through callbacks afaik